python/ray/_private/usage/usage_constants.py

```diff
@@ -46,8 +46,8 @@
 
 LIBRARY_USAGE_PREFIX = "library_usage_"
 
+# Keep in-sync with the same constants defined in usage_stats_client.h
```

Comment:
I think we can move this to protobuf as @scv119 suggested. (no need to take any action from this PR)